test/test-v8debugapi.js

```diff
@@ -28,6 +28,8 @@ var breakpointInFoo = {
   location: { path: 'test-v8debugapi.js', line: 4 }
 };
 
+var MAX_INT = 2147483647;
```

Comment:
Maybe we could rename or comment this since it is not actually the maximum integer in JS.